v20.2/spatial-data.md

```diff
@@ -0,0 +1,305 @@
+toc: true
+---
+
+<span class="version-tag">New in v20.2</span>: CockroachDB has special support for efficiently storing and querying spatial data. Supported spatial data types include:
+
+- Geometric objects such as points, lines, paths, and polygons in 2-dimensional space. These are projected onto the flat surface of a plane and are represented in SQL by the `GEOMETRY` data type.
+
+- Geographic objects, which are also made up of points, lines, polygons, etc., in 2-dimensional space. They are projected onto the surface of a sphere and are represented in SQL by the `GEOGRAPHY` data type. (Technically, they are projected onto a spheroid: "a sphere with a bulge"). This projection means that paths between geographic objects are not straight lines; they are curves, and so the distances between objects are calculated using [great circle math](https://en.wikipedia.org/wiki/Great-circle_distance).
+
+{{site.data.alerts.callout_info}}
+This documentation is for an early alpha release. Be prepared for CockroachDB's spatial support and this documentation to change before the 20.2 GA release.
+{{site.data.alerts.end}}
+
+## Compatibility
+
+Just as CockroachDB strives for [Postgres compatibility](postgresql-compatibility.html), our spatial data support is designed to be as compatible as possible with the functionality provided by the [PostGIS](https://postgis.net) extension.
+
+However, we do not yet implement the full list of PostGIS built-in functions and operators.  Also, our indexing works differently (see the [Performance](#performance) section below).  For a list of the spatial functions we support, see [Geospatial functions](functions-and-operators.html#geospatial-functions).
+
+If your application needs support for functions that are not yet implemented, please check out [our meta-issue for built-in function support on GitHub](https://github.com/cockroachdb/cockroach/issues/49203), which describes how to find an issue for the built-in function(s) you need.
+
+## Troubleshooting
+
+For general CockroachDB troubleshooting information, see [this troubleshooting overview](troubleshooting-overview.html).
+
+If you need help troubleshooting an issue with our spatial support, please get in touch using our [Support resources](support-resources.html).
+
+## Performance
+
+We recommend using `GEOMETRY` as much as possible for best performance, and converting to the `GEOGRAPHY` type only where needed by your application.
+
+In order to avoid full table scans, make sure to add indexes to any columns that are frequently accessed. You can check which queries are using which indexes using the [`EXPLAIN`](explain.html) statement. For more information about general query tuning (including index usage), see [Make queries fast](make-queries-fast.html).
+
+The syntax for adding an (inverted) index to a geometry column is:
+
+{% include copy-clipboard.html %}
+~~~ sql
+CREATE INDEX tornado_geom_idx ON tornadoes USING GIN (geom);
```

Comment:
thought it was using GiST (https://postgis.net/workshops/postgis-intro/indexing.html), maybe GIN works too.

Reply:
Looks like both will work but for now we should say GIN since GIST doesn't work